services/src/messaging/src/main/java/org/openkilda/messaging/info/event/SwitchState.java

```diff
@@ -96,18 +99,11 @@ public String toString() {
         return type;
     }
 
-    public static SwitchState from(String state) {
-        return Arrays.stream(SwitchState.values())
-                .filter(item -> item.type.equalsIgnoreCase(state))
-                .findFirst()
-                .orElseThrow(() -> new IllegalArgumentException(String.format("Incorrect switch state: %s", state)));
-    }
-
     public boolean isActive() {
         return this == ADDED || this == ACTIVATED;
     }
 
     public boolean isInactive() {
```

Comment:
Dangerous combination - 2 method that is supposed to return opposite values, but calculate it in different way. Can we drop one of this(`isActive`, `isInactive`) methods?

Reply:
Good point. I was also thinking about it. In fact `isInactive` method is not used in the project and can be deleted. As you can see, we have 3 states (`CHANGED`, `VALIDATING` & `CACHED`) which not belongs to active or inactive. This is not clear. In general, I can remove `isInactive` method.

Reply:
do it.